### PR TITLE
Allow Symfony 5

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

**Comment by weaverryan** [comment]
Thank you Nicolas!

File: composer.json [hunk 1]
```diff
@@ -19,21 +19,20 @@
             "Symfony\\WebpackEncoreBundle\\Tests\\": "tests/"
         }
     },
+    "minimum-stability": "dev",
```
Comment (B-Galati) [inline composer.json:22]: Did this stay on purpose ?

Comment (nicolas-grekas) [reply composer.json:22]: Yes, it's a best practice to do this for libs.

Comment (B-Galati) [reply composer.json:22]: Thanks! I guess it's because it allows to install unstable deps easily when working on the bundle?

**Pull request merged**